app/templates/partials/answers/currency.html

```diff
@@ -1,3 +1,28 @@
 <div class="field">
-  {{ answer.schema_item.widget.render(answer)|safe}}
+    {% set label = {
+      'for': answer.id,
+      'text': answer.label
+    } %}
+
+    {% include 'partials/forms/label.html' %}
+
+    {% set raw_data = form[answer['id']].raw_data[0] if form[answer['id']].raw_data is not none else '' %}
+    {% set form_data = form[answer['id']].data if form[answer.id].data is not none else '' %}
```

Comment:
Having made default changes quite a few of the integration tests fail. Given I know what a nested nightmare the templates are already, and that they should be refactored over and above this anyway, I'm not going to make this change in this PR as I think it'll be a time suck.